tools/workflow/cleaner/lambda/clean.go

```diff
@@ -79,3 +146,33 @@ func shouldDelete(lf *lambda.FunctionConfiguration) (bool, error) {
 	}
 	return false, nil
 }
+
+func shouldDeleteLayer(layerList *lambda.LayersListItem, expirationDate time.Time) bool {
+	layerARN := layerList.LayerArn
+	regexList := []string{
+		".*:layer:aws-otel-collector.*$",
+		".*:layer:aws-otel-lambda-python.*$",
+		".*:layer:aws-otel-java.*$",
+		".*:layer:aws-otel-lambda-nodejs.*$",
+		".*:layer:aws-otel-go-wrapper.*$",
+		".*:layer:aws-observability.*$",
+		".*:layer:custom-config-layer.*$",
+		".*:layer:NodeProxyAgentLayer.*$",
+		".*:layer:aws-distro-for-opentelemetry.*$",
```

Comment:
What are these and were they created by the lambda integ tests?

Reply:
I am not confident about this Layer - `NodeProxyAgentLayer.` but the rest of them were mostly created during onboarding integ tests etc.,, intitally when the name of the lambda layers are not standardized.

Reply:
Okay, let's remove the `NodeProxyAgentLayer` then.

Reply:
Also, please remove `custom-config-layer`. I see it being actively used in `us-west-2`.

Reply:
Agreed, updated the regex list.